common/supplemental/supplementalData.xml

```diff
@@ -4411,245 +4411,245 @@ XXX Code for transations where no currency is involved
-              <era type="196" start="1596-10-27" code="keicho-1596"/>
-              <era type="197" start="1615-7-13" code="genna-1615"/>
-              <era type="198" start="1624-2-30" code="kanei-1624"/>
-              <era type="199" start="1644-12-16" code="shoho-1644"/>
-              <era type="200" start="1648-2-15" code="keian-1648"/>
-              <era type="201" start="1652-9-18" code="joo-1652"/>
-              <era type="202" start="1655-4-13" code="meireki-1655"/>
-              <era type="203" start="1658-7-23" code="manji-1658"/>
-              <era type="204" start="1661-4-25" code="kanbun-1661"/>
-              <era type="205" start="1673-9-21" code="enpo-1673"/>
-              <era type="206" start="1681-9-29" code="tenna-1681"/>
-              <era type="207" start="1684-2-21" code="jokyo-1684"/>
-              <era type="208" start="1688-9-30" code="genroku-1688"/>
-              <era type="209" start="1704-3-13" code="hoei-1704"/>
-              <era type="210" start="1711-4-25" code="shotoku-1711"/>
-              <era type="211" start="1716-6-22" code="kyoho-1716"/>
-              <era type="212" start="1736-4-28" code="genbun-1736"/>
-              <era type="213" start="1741-2-27" code="kanpo-1741"/>
-              <era type="214" start="1744-2-21" code="enkyo-1744"/>
-              <era type="215" start="1748-7-12" code="kanen-1748"/>
-              <era type="216" start="1751-10-27" code="horeki-1751"/>
-              <era type="217" start="1764-6-2" code="meiwa-1764"/>
-              <era type="218" start="1772-11-16" code="anei-1772"/>
-              <era type="219" start="1781-4-2" code="tenmei-1781"/>
-              <era type="220" start="1789-1-25" code="kansei-1789"/>
-              <era type="221" start="1801-2-5" code="kyowa-1801"/>
-              <era type="222" start="1804-2-11" code="bunka-1804"/>
-              <era type="223" start="1818-4-22" code="bunsei-1818"/>
-              <era type="224" start="1830-12-10" code="tenpo-1830"/>
-              <era type="225" start="1844-12-2" code="koka-1844"/>
-              <era type="226" start="1848-2-28" code="kaei-1848"/>
-              <era type="227" start="1854-11-27" code="ansei-1854"/>
-              <era type="228" start="1860-3-18" code="manen-1860"/>
-              <era type="229" start="1861-2-19" code="bunkyu-1861"/>
-              <era type="230" start="1864-2-20" code="genji-1864"/>
-              <era type="231" start="1865-4-7" code="keio-1865"/>
-              <era type="232" start="1868-9-8" code="meiji" aliases="m"/>
-              <era type="233" start="1912-7-30" code="taisho" aliases="t"/>
-              <era type="234" start="1926-12-25" code="showa" aliases="s"/>
-              <era type="235" start="1989-1-8" code="heisei" aliases="h"/>
-              <era type="236" start="2019-5-1" code="reiwa" aliases="r"/>
+              <era type="-2" start="645-6-19" code="japanese-inverse" aliases="gregory-inverse bc bce"/>
```

Comment:
Question; wouldn't the japanese-inverse end on that date (or actually the day before)?

Reply:
?

Reply:
Fixed.